system-int-test/test/cmr/system_int_test/search/facets/granule_facets_v2_search_test.clj

```diff
@@ -0,0 +1,56 @@
+(ns cmr.system-int-test.search.facets.granule-facets-v2-search-test
+  "This tests retrieving v2 facets when searching for granules."
+  (:require
+   [clojure.test :refer :all]
+   [cmr.common.util :as util]
+   [cmr.common-app.services.search.query-validation :as cqv]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.search-util :as search]))
+
+(defn- search-and-return-v2-facets
+  "Returns only the facets from a v2 granule facets search request."
+  ([]
+   (search-and-return-v2-facets {}))
+  ([search-params]
+   (index/wait-until-indexed)
+   (let [query-params (merge search-params {:page-size 0 :include-facets "v2"})]
+     (get-in (search/find-concepts-json :granule query-params) [:results :facets]))))
+
+(deftest granule-facet-tests
+  (testing "Granule facets are returned when requesting V2 facets in JSON format."
+    (let [facets (search-and-return-v2-facets)]
+      (is (= "Browse Granules" (:title facets)))))
+
+  (testing "Only the json format supports V2 granule facets."
+    (let [unsupported-formats (remove #(or (= :timeline %) (= :json %))
```

Comment:
Why do we remove `:timeline` format from the test? Does it return a different error message?
The remove pred check can be done as `#(#{:timeline :json} %)` which is more readable than `or`.

Reply:
I was gonna ask the same.

Reply:
Don't tell me #(#{:timeline :json} %) is more readable than anything! 😆 

I removed timeline because we don't support .timeline as an extension and I couldn't be bothered to figure out how to test that it also returns the correct message. Now that I've been called out, I'll take a look to see if I can test it, or worst case add a comment.

Reply:
What is #(#{:timeline :json} %)? Stay with "or" please :)